package.json

```diff
@@ -9,6 +9,7 @@
     "react": "15.3.2",
     "react-native": "0.34.0",
     "react-native-button": "^1.7.0",
+    "react-native-camera": "git+https://github.com/lwansbrough/react-native-camera.git",
```

Comment:
http://stackoverflow.com/a/14188105/3133680
is the closest thing I found that would help us.

Will try it out later, but I don't think it's that big a deal for now.